runtime/lua/vim/lsp/util.lua

```diff
@@ -1757,37 +1757,56 @@ end
 ---
 ---@param symbols lsp.DocumentSymbol[]|lsp.SymbolInformation[]
 ---@param bufnr? integer
+---@param offset_encoding? 'utf-8'|'utf-16'|'utf-32'
+---                         default to first client of buffer
 ---@return vim.quickfix.entry[] # See |setqflist()| for the format
-function M.symbols_to_items(symbols, bufnr)
+function M.symbols_to_items(symbols, bufnr, offset_encoding)
   bufnr = bufnr or 0
+  if offset_encoding == nil then
+    vim.notify_once(
+      'symbols_to_items must be called with valid offset encoding',
+      vim.log.levels.WARN
+    )
+    local clients = vim.lsp.get_clients({ bufnr = bufnr })
```

Comment:
If `clients` is empty an early return feels cleaner than setting `offset_encoding` to `utf-16`.

Reply:
In the previous API, this function could only pass `symbols`, and I am concerned that some people might use it solely to convert `symbols` (although in nvim's code, it is only used for `vim.lsp.buf.document_symbol`).

Reply:
Hmmm. That feels strange. But if we decide to support this use, I think we should mention that `offset_encoding` will be `utf-16` if there are no clients in the buffer.

Reply:
I think we haven't resolved this yet. If `clients` is empty here it seems like a misuse of this function?